src/main/java/org/folio/rest/impl/DataExportImpl.java

```diff
@@ -64,13 +64,18 @@ public void postDataExportExport(ExportRequest entity, Map<String, String> okapi
         jobProfileService.getById(entity.getJobProfileId(), tenantId)
           .onSuccess(jobProfile ->
             mappingProfileService.getById(jobProfile.getMappingProfileId(), tenantId)
-              .onSuccess(mappingProfile -> {
-                inputDataManager.init(JsonObject.mapFrom(entity), JsonObject.mapFrom(requestFileDefinition), JsonObject.mapFrom(mappingProfile), okapiHeaders);
-                succeededFuture()
-                  .map(PostDataExportExportResponse.respond204())
-                  .map(Response.class::cast)
-                  .onComplete(asyncResultHandler);
-              })
+              .onSuccess(mappingProfile ->
+                jobExecutionService.getById(requestFileDefinition.getJobExecutionId(), tenantId)
+                  .onSuccess(jobExecution ->
+                    jobExecutionService.updateJobExecutionWithJobProfileInfo(jobExecution, jobProfile, tenantId)
```

Comment:
The 2nd option you mentioned in the approach is actually good enough. We need not populate the job profile name and store it in DB while exporting, just the ID is enough,

Reply:
I did not update the job execution while calling get job execution by id, to reduce interactions to DB. The only place where we are populating jobProfileName and updating the object in DB it is here, so I think we need to leave it as is here

Reply:
I don`t update the jobExecution object in DB while calling get job execution, to reduce interactions to the DB. The only one place where we are set the jobProfileName to the jobExecution and update the object in the DB is during export, so I think we should leave this logic as is.

Reply:
why can`t we leave it as is? if we will remove set jobProfileName here, all jobExecutions in the DB will not have this field